communication/data_logging.cpp

```diff
@@ -344,49 +342,48 @@ bool Data_logging::checksum_control(void)
 {
     bool new_values = false;
 
-    double cksum_a_current = 0.0;
-    double cksum_b_current = 0.0;
+    double cksum_a__current = 0.0;
```

Comment:
Oups,
it is the auto modification of class members adding them a '_' at the end that mess-up.
sorry